examples/routing_service/routing_service_file_adapter/cpp/CMakeLists.txt

```diff
@@ -45,6 +45,19 @@ target_link_libraries(
 
 # Copy the sample data file into the binary directory
 configure_file(
```

Comment:
Actually we should be using a `custom_command` to copy the files

Reply:
Is there a reason behind this preference? Any best practices?

Reply:
I think this post can be useful: https://dev.to/iblancasa/learning-cmake-3-understanding-addcustomcommand-and-addcustomtarget-43gp

The reason is that `configure_file` will be run during the build system creation. Probably, it will make more sense to do the copy of the file during the build. But in this case, is up to you :) .

Reply:
Hey @iblancasa good job with the article. That was an interesting read. However this example doesn't require all the intricacies you described. Sin the CSV and XML files are typically immutable, we should be okay with configure_file().